fn_cve_search/tox.ini

```diff
@@ -0,0 +1,17 @@
+# Tox (https://tox.readthedocs.io/) is a tool for running tests
```

Comment:
This file is not needed as there are no unit tests...

Reply:
file removed.